homeassistant/components/homekit/type_lights.py

```diff
@@ -0,0 +1,209 @@
+"""Class to hold all light accessories."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_RGB_COLOR, ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS, SUPPORT_RGB_COLOR)
+from homeassistant.const import ATTR_SUPPORTED_FEATURES, STATE_ON, STATE_OFF
+
+from . import TYPES
+from .accessories import HomeAccessory, add_preload_service
+from .const import (
+    CATEGORY_LIGHT, SERV_LIGHTBULB,
+    CHAR_BRIGHTNESS, CHAR_HUE, CHAR_ON, CHAR_SATURATION)
+
+_LOGGER = logging.getLogger(__name__)
+
+RGB_COLOR = 'rgb_color'
+
+
+class Color:
```

Comment:
Why not use functions in util/color.py?

Reply:
Missed that. I might have merge this one a bit to quickly